pkg/features/kube_features.go

```diff
@@ -28,24 +28,29 @@ import (
 	kcmfeatures "k8s.io/controller-manager/pkg/features"
 )
 
+// Every feature gate should add method here following this template:
+//
+// // owner: @username
+// // kep: https://kep.k8s.io/NNN
+// MyFeature featuregate.Feature = "MyFeature"
+//
+// Feature gates should be listed in alphabetical, case-sensitive
+// (upper before any lower case character) order. This reduces the risk
+// of code conflicts because changes are more likely to be scattered
+// across the file.
 const (
-	// Every feature gate should add method here following this template:
```

Comment:
Do we want to fix the "add method here", now that the text is being touched?

Something like "Every feature gate should have an entry here following. this template:".

Same elsewhere.

Reply:
Done